src/ai/susi/server/api/aaa/RemoveUserDevices.java

```diff
@@ -75,13 +75,19 @@ public JSONObject getDefaultPermissions(UserRole baseUserRole) {
     }
 
     @Override
-    public ServiceResponse serviceImpl(Query query, HttpServletResponse response, Authorization authorization, JsonObjectWithDefault permissions) throws APIException {
+    public ServiceResponse serviceImpl(Query query, HttpServletResponse response, Authorization rights, JsonObjectWithDefault permissions) throws APIException {
```

Comment:
Do not change naming convention here. Instead, change variable name at the other instance.